### DISCUSSION TITLE
export/import of accounts configuration

### DISCUSSION THREAD

**Discussion by strk** [cheeaun/phanpy#358] [Ideas]
I've been testing phanpy from the phanpy.social URL and now would like to have it served by my own server. Everytime I move the URL the accounts configuration is lost. I do understand this is a browser-side thing but wonder if phanpy could make migrating the config from one place to another easier by implementing an import/export function for those settings.

**Comment by cheeaun** [comment]
This is also in my to-do list, though longer-term solution would be this https://github.com/mastodon/mastodon/issues/28265
